guides/source/debugging_rails_applications.md

```diff
@@ -728,10 +752,11 @@ No breakpoints.
 
 You can also enable or disable breakpoints:
 
-* `enable breakpoints`: allow a _breakpoints_ list or all of them if no list is
-specified, to stop your program. This is the default state when you create a
+* `enable breakpoints[ n[ m[ ...]]]`: allows a specific breakpoint list or all
```

Comment:
> enable breakpoints [n[ m[ ...]]]

@deivid-rodriguez I see you changed the spacing elsewhere as well(before n, instead of after breakpoints), is that intentional?